lib/Jsrt/Jsrt.cpp

```diff
@@ -3770,42 +3733,45 @@ CHAKRA_API JsTTDRawBufferAsyncModifyComplete(_In_ byte* finalModPos)
 #endif
 }
 
+CHAKRA_API JsTTDCheckAndAssertIfTTDRunning(_In_ const char* msg)
```

Comment:
This seems like a weird API to expose to someone embedding chakracore- can we make this debug only at least? It would be better if we figure out a way without exposing this as a Jsrt API